test/sql/test_array_fn/R/test_array_fn

```diff
@@ -3656,17 +3616,14 @@ E: (1064, 'Getting analyzing error from line 1, column 7 to line 1, column 34. D
 -- !result
 select array_intersect(aad_1, as_1) from array_test order by pk;
 -- result:
-[]
-None
-None
-[]
+E: (1064, 'Getting analyzing error from line 1, column 7 to line 1, column 34. Detail message: No matching function with signature: array_intersect(array<array<array<decimal128(26, 2)>>>, array<array<varchar(65533)>>).')
 -- !result
 select array_intersect(aad_1, aas_1) from array_test order by pk;
 -- result:
-[[[1.00],[2.00],[3.00]],[[6.00],[5.00],[4.00]],[[-1.00,-2.00],[-2.00,10.00],[100.00,23.00]]]
-[[[10.00],[20.00],[30.00]]]
+[]
+[]
 None
-[[[6.00],[5.00],[4.00]]]
+[]
```

Comment:
why?

Reply:
`array<decimal>` cast to `array<string>` and has a number point str....
like `'1.00'` can't match `'1'`